addons/account/views/tests_shared_js_python.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <template id="tests_shared_js_python">
+        <t t-call="web.frontend_layout">
+            <t t-set="no_header" t-value="True"/>
+            <t t-set="no_footer" t-value="True"/>
+            <owl-component name="account.tests_shared_js_python"/>
```

Comment:
Note: I tried to use assets_backend instead and create the component like:
https://www.odoo.com/documentation/17.0/developer/howtos/standalone_owl_application.html

However, I'm not longer to initiate the owl component that way. Indeed, the component is always trying to use some services like menu/translation etc and there are not there for some reason...